tests/decodeinputavformat.cpp

```diff
@@ -53,7 +59,11 @@ bool DecodeInputAvFormat::initInput(const char* fileName)
             if (ctx->extradata && ctx->extradata_size)
                 m_codecData.append((char*)ctx->extradata, ctx->extradata_size);
             m_videoId = i;
-            setResolution(ctx->coded_width, ctx->coded_height);
+            if (ctx->coded_width > width || ctx->height > height) {
```

Comment:
using "||" may set a larger width and a smaller height. Would better judge height and width independent.